packages/common/src/relay/helpers.ts

```diff
@@ -54,7 +54,11 @@ export const reduceConnectParams = (peers: DeepArray<IMakeCallParams>, callDevic
           }
         }
       } else if (type === 'sip') {
-        tmp = { type, params: { to, from: from || defaultFromNumber } }
+        const { headers, codecs, webrtc_media } = (peer as MakeSipCallParams);
```

Comment:
I think you can remove the `as MakeSipCallParams` (and the import) as we discussed yesterday?